README.md

```diff
@@ -30,7 +30,12 @@
 
```

Comment:
- [ ] It would be nice if you fix your first two queries so their execution time should be less than 1000ms because 1000ms is considered a lot of time so by standard your execution time should be less than 1000ms 👍